apache/incubator-zeppelin
# [ZEPPELIN-575] Cassandra interpreter V2 documentation

### What is this PR for?
Update official documentation for **Cassandra interpreter V2**
Related to PR: https://github.com/apache/incubator-zeppelin/pull/600

### What type of PR is it?
[Documentation]

### Is there a relevant Jira issue?
**[ZEPPELIN-575]** 

### How should this be tested?
Just check the doc after the build to see if you can see the **chapter 14 Change Log**

### Questions:
* Does the licenses files need update? --> **No**
* Is there breaking changes for older versions? --> **No**
* Does this needs documentation? --> **No**

[ZEPPELIN-575]: https://issues.apache.org/jira/browse/ZEPPELIN-575
